### PR TITLE
Added next-nprogress-bar package for a loading state during route change

### PR DESCRIPTION
**Pull request by kelechukwu1**
## Description

Please include a summary of the change and which issue is fixed (if applicable).

## Related Issue

- Fixes # (issue number)

## Contribution Guidelines

Before submitting this pull request, please review our [Contribution Guidelines](https://github.com/abeg-help/frontend/blob/dev/CONTRIBUTING.md) to understand how to contribute to this project.

## Checklist

- [x] I have reviewed the Contribution Guidelines linked above.
- [x] I have tested my changes thoroughly and ensured that all existing tests pass.
- [x] I have provided clear and concise commit messages.
- [ ] I have updated the project's documentation as necessary.

### CODE REVIEW

File: components/primitives/abeg-help.me/abeg-help-me-svg.tsx
Comment (289Volts) [inline components/primitives/abeg-help.me/abeg-help-me-svg.tsx:1]: Please pull latest code from the dev branch and check the components folder for the logoBanner component. It receives a prop for changing the text-color

File: package.json [hunk 1]
```diff
@@ -33,6 +33,7 @@
     "@tanstack/react-query-devtools": "^5.8.1",
     "clsx": "^2.0.0",
     "next": "13.5.4",
+    "next-nprogress-bar": "^2.1.2",
```
Comment (289Volts) [inline package.json]: please uninstall this package. I'm sure our progress bar component can handle this

File: pages/reset-password/confirmation.tsx [hunk 2]
```diff
@@ -1,52 +1,56 @@
...
-              stroke="#007004"
-              strokeWidth="5"
-              strokeLinecap="round"
-              strokeLinejoin="round"
-            />
-          </svg>
+      <div className="mx-auto w-full bg-white p-6 rounded-md justify-center items-center md:w-1/2 lg:w-4/12 space-y-10">
+        <div>
+          <div className="flex justify-center items-center mb-8">
+            <svg
```
Comment (289Volts) [inline pages/reset-password/confirmation.tsx]: Also, please create svg as files inside the assets folder in public folder. If it's an image, it'll be in images, if it's an icon, it'll be in icons

**Review by 289Volts — CHANGES_REQUESTED** [review]
Please check the comments i made on the files that require changes

File: package.json (hunk 1 above)
Comment (kelechukwu1) [reply package.json]: dode

File: components/primitives/abeg-help.me/abeg-help-me-svg.tsx
Comment (kelechukwu1) [reply components/primitives/abeg-help.me/abeg-help-me-svg.tsx:1]: done

**Review by 289Volts — APPROVED** [review]
Good job. Thank you for taking the honest feedback and making the necessary changes

**Pull request merged**